Documentation/DocuBlocks/Rest/Graph Traversal/HTTP_API_TRAVERSAL.md

```diff
@@ -100,8 +100,7 @@ item iteration order can be *"forward"* or *"backward"*
 
 @RESTBODYPARAM{uniqueness,string,optional,string}
 specifies uniqueness for vertices and edges visited.
-If set, must be an object like this:
-
+If set, must be an object like this:<br>
 `"uniqueness": {"vertices": "none"|"global"|"path", "edges": "none"|"global"|"path"}`
```

Comment:
No change in Swagger UI, but fixes docu rendering. Before:

![image](https://user-images.githubusercontent.com/7819991/69811335-26db0d00-11ee-11ea-8c21-ca05d716ffe2.png)

After:

![image](https://user-images.githubusercontent.com/7819991/69811357-322e3880-11ee-11ea-95e6-53acca1b2795.png)